include/mbedtls/ssl.h

```diff
@@ -801,6 +804,12 @@ typedef struct mbedtls_ssl_key_cert mbedtls_ssl_key_cert;
 typedef struct mbedtls_ssl_flight_item mbedtls_ssl_flight_item;
 #endif
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && \
+    defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C)
```

Comment:
```suggestion
#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C)
```
as MBEDTLS_SSL_EARLY_DATA cannot be defined if MBEDTLS_SSL_EARLY_DATA is not. Please change all similar instances.